### PR TITLE
Add an integration test for schema watch

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

**Review by vroldanbet — COMMENTED** [review]
@josephschorr changes look good to me, although it does not seem like there is an assertion for actual schema watch logs

File: cmd/spicedb/schemawatch_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	}
+	require.Equal(t, len(engines), len(datastore.Engines))
+
+	for driverName, shouldRun := range engines {
+		if !shouldRun {
+			continue
+		}
+
+		t.Run(driverName, func(t *testing.T) {
+			bridgeNetworkName := fmt.Sprintf("bridge-%s", uuid.New().String())
```
Comment (vroldanbet) [inline cmd/spicedb/schemawatch_integration_test.go:36]: seem like expensive tests, let's parallelize them

```suggestion
			t.Parallel()
			
			bridgeNetworkName := fmt.Sprintf("bridge-%s", uuid.New().String())
```

File: cmd/spicedb/schemawatch_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	testdatastore "github.com/authzed/spicedb/internal/testserver/datastore"
+	"github.com/authzed/spicedb/pkg/datastore"
+)
+
+func TestSchemaWatch(t *testing.T) {
+	engines := map[string]bool{
+		"postgres":    false,
+		"mysql":       false,
+		"cockroachdb": true,
+		"spanner":     false,
```
Comment (vroldanbet) [inline cmd/spicedb/schemawatch_integration_test.go:26]: did you mean to disable postgres and spanner?

File: cmd/spicedb/schemawatch_integration_test.go (hunk 1 above)
Comment (josephschorr) [reply cmd/spicedb/schemawatch_integration_test.go:36]: I don't like running datastore container tests in parallel; things to get wonky when we do that

File: cmd/spicedb/schemawatch_integration_test.go (hunk 2 above)
Comment (josephschorr) [reply cmd/spicedb/schemawatch_integration_test.go:26]: Yes

**Review by vroldanbet — APPROVED** [review]

**Pull request merged**